core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -103,7 +103,7 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
       val taskSortColumn = Option(parameterTaskSortColumn).map { sortColumn =>
         UIUtils.decodeURLParameter(sortColumn)
       }.getOrElse("Index")
-      val taskSortDesc = Option(parameterTaskSortDesc).map(_.toBoolean).getOrElse(false)
+      val taskSortDesc = Option(parameterTaskSortDesc).exists(_.toBoolean)
```

Comment:
yes, I meant the question is whether .exists is in 2.10, but I am back at a desktop now and found it is, so no problem. That's not the issue, just, whether it's clearer